academy/ibc/ibc-tao-dev.md

```diff
@@ -0,0 +1,783 @@
+---
+title: "IBC/TAO"
+order: 
+description: 
+tag: deep-dive
+---
+
+# Transport, Authentication, and Ordering Layer
+
+Now that you covered the introduction and have a better understanding of how different Inter-Blockchain Communication (IBC) Protocol components and Interchain Standards (ICS) relate to each other, take a deep dive into IBC/TAO (transport, authentication, and ordering) and the [IBC module](https://github.com/cosmos/ibc-go).
+
+## Connections
+
```

Comment:
would be good to offer more context about why we have pairwise connections:

from the internal recording: Connections under ICS3 allow us to make sure that there isn't a malicious entity who can come in pretending to be a counter party chain and forge incorrect information. It also allows us to establish these connections with on-chain ledger code without us having to rely on off-chain processes